src/nectarbase/objects.py

```
@@ -27,15 +27,21 @@
     )
 
 
-class Amount(object):
-    def __init__(self, d, prefix=default_prefix, json_str=False):
+class Amount:
+    def __init__(
+        self,
+        d: Union[str, Any],
+        prefix: str = default_prefix,
+        json_str: bool = False,
+        **kwargs,
```

Ruff (0.14.8)

Unused method argument: `kwargs`

(ARG002)